src/test/common/test_crc32c.cc

```diff
@@ -265,3 +265,44 @@ TEST(Crc32c, RangeNull) {
     ASSERT_EQ(crc, *check);
   }
 }
+
+TEST(Crc32c, zeros_performance_compare) {
+  utime_t start;
+  utime_t end;
+
+  for (size_t scale=1; scale < 31; scale++)
+  {
+    size_t size = (1<<scale) + rand()%(1<<scale);
+    start = ceph_clock_now();
+    uint32_t crc_a = ceph_crc32c(111, nullptr, size);
+    end = ceph_clock_now();
+    std::cout << "regular  method. size=" << size << " time= " << (double)(end-start)
+        << " at " << (double)size/(1024*1024)/(double)(end-start) << " MB/sec" << std::endl;
+
+    start = ceph_clock_now();
+    uint32_t crc_b = ceph_crc32c_sctp(111, nullptr, size);
+    end = ceph_clock_now();
+    std::cout << "fallback method. size=" << size << " time=" << (double)(end-start)
+        << " at " << (double)size/(1024*1024)/(double)(end-start) << " MB/sec" << std::endl;
+
+    EXPECT_EQ(crc_a, crc_b);
+  }
+}
+
+TEST(Crc32c, zeros_performance) {
+  constexpr size_t ITER=100000;
+  utime_t start;
+  utime_t end;
+
+  start = ceph_clock_now();
+  for (size_t i=0; i<ITER; i++)
+  for (size_t scale=1; scale < 31; scale++)
+  {
+    size_t size = (1<<scale) + rand() % (1<<scale);
+    ceph_crc32c(rand(), nullptr, size);
```

Comment:
@kestrels 
1) I added estimation of error in time measurement.
2) I kept using clock_gettime, which has resolution 1e-9s on linux x86